io-ballerina/build.gradle

```diff
@@ -245,4 +248,13 @@ ballerinaBuild.dependsOn updateTomlFile
 ballerinaBuild.finalizedBy revertTomlFile
 build.dependsOn ballerinaBuild
 
+ballerinaPublish.dependsOn ":${packageName}-native:build"
+ballerinaBuild.dependsOn ":${packageName}-native:test"
+ballerinaBuild.dependsOn ":${packageName}-test-utils:build"
+ballerinaPublish.dependsOn ballerinaBuild
+ballerinaPublish.dependsOn updateTomlFile
+ballerinaPublish.dependsOn initializeVariables
+ballerinaPublish.finalizedBy revertTomlFile
+publish.dependsOn ballerinaPublish
```

Comment:
```suggestion
ballerinaBuild.dependsOn ":${packageName}-native:test"
ballerinaBuild.dependsOn ":${packageName}-test-utils:build"

ballerinaPublish.dependsOn ":${packageName}-native:build"
ballerinaPublish.dependsOn ballerinaBuild
ballerinaPublish.dependsOn updateTomlFile
ballerinaPublish.dependsOn initializeVariables
ballerinaPublish.finalizedBy revertTomlFile
publish.dependsOn ballerinaPublish
```

Reply:
@aashikam 

```
ballerinaBuild.dependsOn ":${packageName}-native:test"
ballerinaBuild.dependsOn ":${packageName}-test-utils:build"
```
 is alreadly there.

Reply:
I just reordered it.

Reply:
```
ballerinaBuild.dependsOn ":${packageName}-native:test"
ballerinaBuild.dependsOn ":${packageName}-test-utils:build"
``` were among `ballerinaPublish` tasks. I took them out. For more readability